src/applications/disability-benefits/all-claims/config/form0781/index.js

```diff
@@ -13,4 +19,18 @@ export const form0781PagesConfig = {
     uiSchema: workflowChoicePage.uiSchema,
     schema: workflowChoicePage.schema,
   },
+  manualUploadPage: {
+    path:
+      'disability/file-disability-claim-form-21-526ez/additional-forms/mental-health-statement/upload',
+    uiSchema: manualUploadPage.uiSchema,
+    depends: formData => showManualUpload0781Page(formData),
+    schema: manualUploadPage.schema,
+  },
+  mentalHealthSupportInfoPage: {
```

Comment:
Potentially take out if Scott's merges first (https://github.com/department-of-veterans-affairs/va.gov-team/issues/97078)

Reply:
Because he will have this page actually implemented

Reply:
TODO: do unit tests for these: **showManualUpload0781Page(formData),**